docs/docs/Sample-Flows/financial-report-parser.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Financial report parser
+slug: /financial-report-parser
+---
+
+import Icon from "@site/src/components/icon";
+
+Build a **Financial Report Parser** flow with the [Structured output](/components-helpers#structured-output) and [Parser](/components-processing#parser) components to parse LLM responses into a structured format.
+
+In this example, the **Chat Input** component is pre-loaded with a sample financial report to demonstrate extracting `Gross Profit`, `EBITDA`, and `Net Income`.
+
+## Prerequisites
+
+- [A running Langflow instance](/get-started-installation)
+- [An OpenAI API key](https://platform.openai.com/)
+
+## Create the financial report parser flow
+
+1. From the Langflow dashboard, click **New Flow**.
+2. Select **Financial Report Parser**.
+3. The **Financial Report Parser** flow is created.
```

Comment:
This should not be a numbered step because the user doesn't have to take any action.